docs/RELEASE_NOTES.md

```diff
@@ -55,13 +55,14 @@
 **Bug Fixes** (1)
 - **Commons** - Fixed library version in User-Agent headers of JVM SDK clients using the library. [#191](https://github.com/commercetools/commercetools-sync-java/issues/191)
 
-**Migration guide** (4)
+**Migration guide** (6)
+- **Product Sync** - Removed `removeOtherVariants` option which is already done by the sync by default. [#26](https://github.com/commercetools/commercetools-sync-java/issues/26)
+- **Commons** - Removed `removeOtherSetEntries`, `removeOtherCollectionEntries` and `removeOtherProperties`. [#26](https://github.com/commercetools/commercetools-sync-java/issues/26)
```

Comment:
Same a above

Reply:
c2dca78 I added the [KeepOtherVariantsSync](https://github.com/commercetools/commercetools-sync-java/pull/196/files#diff-8b48435be042780bac4308837e491ea2) as a template and an [integration test](https://github.com/commercetools/commercetools-sync-java/pull/196/files#diff-c2377e77a323178f42915dc71af1a73d) for it that the release notes [links to](c2dca78).